templates/filebeat-5.yml

```diff
@@ -16,6 +16,9 @@ filebeat:
       scan_frequency: 10s
       harvester_buffer_size: {{ harvester_buffer_size }}
       max_bytes: {{ max_bytes }}
+      ignore_older: {{ ignore_older }}
+      clean_inactive: {{ clean_inactive }}
+      close_timeout: {{ close_timeout }}
```

Comment:
can we do some sanity checking of these values? Passing them verbatim will lead to breakage if the user configures them incorrectly

Reply:
I guess we should block the unit and throw an error if a sanity check on config values fails. Thoughts?

Reply:
yes, setting the charm to blocked until the configuration is amended seems reasonable to me

Reply:
I have resubmitted the patch with sanity checks for the values, the unit will get into blocked statem if the checks fail and will log extra details on why it failed.